### PR TITLE
movie api ruby

### PR DESCRIPTION
**Pull request by Evvn**

### CODE REVIEW

File: Evan/wk05/1-mon/movies/main.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+require 'sinatra/reloader'
+require 'HTTParty'
+
+get '/' do
+  erb :index
+end
+
+get '/movie_get' do
+  @movie_name = params[:movie_name]
+  @movie = HTTParty.get("http://omdbapi.com/?t=#{ @movie_name.downcase }&apikey=2f6435d9")
```
Comment (Sonna) [inline Evan/wk05/1-mon/movies/main.rb]: **OH NO**, not the API key!
scream

Comment (Sonna) [reply Evan/wk05/1-mon/movies/main.rb]: I retract my earlier comment, all students seem to be doing this. 😐

Comment (Evvn) [reply Evan/wk05/1-mon/movies/main.rb]: shrug

Comment (Evvn) [reply Evan/wk05/1-mon/movies/main.rb]: screen shot 2018-07-17 at 10 40 30 am

@kasun-maldeni

Comment (kasun-maldeni) [reply Evan/wk05/1-mon/movies/main.rb]: I would recommend putting the API key in an environment variable. We'll talk more about this later in the course.

**Pull request merged**